### ISSUE TITLE
INCLUDE for indexes isn't supported

### ISSUE THREAD

**Issue by YorickPeterse** [pganalyze/pg_query#184]
The following SQL query is valid starting with PG 11 (see https://www.postgresql.org/docs/11/sql-createindex.html), but doesn't parse with version 1.3.0 of this Gem:

```ruby
PgQuery.normalize('create index concurrently foo on test (bla) include (bar)')
```

This produces the following error:

```
PgQuery::ParseError: syntax error at or near "include" (scan.l:1158)
```

Looking at https://github.com/lfittl/pg_query/blob/master/ext/pg_query/extconf.rb#L7, this seems to happen because the Gem builds itself against PG 10, which doesn't yet support this syntax.

**Comment by stanhu** [comment]
It sounds like this will be resolved once PostgreSQL 12 support is shipped: https://github.com/lfittl/libpg_query/pull/67

**Comment by lfittl** [comment]
@YorickPeterse As @stanhu notes, that will be resolved once the parser is updated (https://github.com/lfittl/pg_query/pull/187 is the correct tracking PR for that).

**Comment by lfittl** [comment]
@YorickPeterse And this is released now as `2.0.0`, thus closing the issue. Thanks for the report!

```
irb(main):001:0> PgQuery.normalize('create index concurrently foo on test (bla) include (bar)')
=> "create index concurrently foo on test (bla) include (bar)"
```

**Comment by YorickPeterse** [comment]
@lfittl Thanks!
